### PR TITLE
Add containerd to image

### PR DESCRIPTION
**Pull request by muhamadazmy**

### CODE REVIEW

File: internals/containerd.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    fi
+}
+
+prepare_containerd() {
+    echo "[+] loading source code: containerd"
+    github_force containerd/containerd $CONTAINERD_VERSION containerd
+}
+
+compile_containerd() {
+    echo "[+] compiling coreX and core0"
```
Comment (zaibon) [inline internals/containerd.sh]:
```suggestion
    echo "[+] compiling containerd"
```

**Comment by muhamadazmy** [comment]
Please do not merge, since we need to add `runc` too to the image.

File: internals/containerd.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
+CONTAINERD_VERSION="v1.2.7"
+
+github_force() {
```
Comment (maxux) [inline internals/containerd.sh]: Same as runc.sh

Comment (muhamadazmy) [reply internals/containerd.sh]: I based my code on the cores.sh builder and it defines it's own `github_force` function.

**Comment by muhamadazmy** [comment]
@maxux I applied your comments. Is it okay to merge this branch now ?

**Comment by muhamadazmy** [comment]
@zaibon they are part of containerd and they are built and copied to the image
image

**Pull request merged**